docs/repositories-configure/configuring-code-patterns.md

```diff
@@ -1,57 +1,48 @@
 
-    -   If your repository is following an [organization coding standard](../organizations/using-coding-standards.md), changes made to any tool or code pattern cause the repository to stop following the coding standard. In this case Codacy asks for your confirmation before accepting the changes, and then copies the coding standard configurations to your repository so you can customize them.
+## Configuring tools and code patterns using the Codacy UI
 
-To configure the tools and code patterns for your repository:
+!!! note
+    If you update the configurations of a repository that follows a [coding standard](../organizations/using-coding-standards.md), Codacy copies the coding standard configurations to the repository and the repository stops following the coding standard. You can then customize the repository configurations without affecting the coding standard.
+
+To configure the tools and code patterns for a repository using the Codacy UI:
 
 1.  Open your repository **Code patterns** page.
 
     ![Code patterns page](images/code-patterns.png)
 
-1.  Toggle the tools that Codacy will use to analyze your repository.
+1.  Enable or disable the tools that Codacy should use to analyze the repository.
 
     ![Toggling tools](images/code-patterns-toggle-tools.png)
 
-1.  Select each tool to configure and toggle the corresponding code patterns using the checkbox next to each pattern.
-
-    !!! tip
-        Codacy displays the tag **New** for one month next to the name of any recently added code patterns.
+1.  Select a tool to enable or disable its code patterns.
 
-    ![Configuring code patterns](images/code-patterns-configure.png)
+    To filter the patterns by language, category, or status, use the sidebar filters.
 
     To see an explanation of the issues that a pattern detects and how to fix them, click **Show details**. Some patterns also allow you to configure the rules for detecting issues.
 
-    ![Code pattern details](images/code-patterns-detail.png)
-
-1.  Optionally, [manually reanalyze your repository](../faq/repositories/how-do-i-reanalyze-my-repository.md) to immediately take the changes into account. Otherwise, Codacy will use the new configuration when it analyzes a new commit or pull request.
-
-## Pattern filters
-
-To make it easier to find relevant code patterns, you can use the sidebar to filter the patterns by language (only for tools that support multiple languages), category, or status:
-
-![Filtering code patterns](images/code-patterns-filter.png)
+    !!! tip
+        -   To enable a group of code patterns, use the filter to select the relevant group of patterns and click **Enable all**.
 
-!!! tip
-    To enable **all code patterns** matching a specific category, use the filter to select the relevant patterns and click **Enable all**.
+            For example, to enable all Security patterns, click the **Security** filter and then click **Enable all**.
```

Comment:
Please evaluate if this example fits ok on the above line, right after the first sentence.
It would shorten the tip. 😉

Reply:
Yes, it can. This is a style choice that we can discuss: I have the habit of leaving lots of white space and paragraphs but may overdo it sometimes.